.github/workflows/gen_gbs_result_badge.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+##
+# SPDX-License-Identifier: LGPL-2.1-only
+# Copyright (C) 2024 Samsung Electronics
+#
+# @file gen_gbs_result_badge.py
+# @brief A tool for generating a github badge image in svg format
+# @author Gichan Jang <gichan2.jang@samsung.com>
+#
+import sys
+import os
+import json
+from pybadges import badge
+
+##
+# @brief Save the badge on the given path
+# @param[in] path A file path to save the svg file
+# @param[in] badge github badge svg file
+def save_badge (path, badge):
+    file = open(path, "w")
+    file.write(badge)
+    file.close()
+
```

Comment:
Thank you for always working hard for the part. 👍 
Thanks for the great PR.

I think it would be good to modify it like this. How about you?
First, ```file.colse()``` can be handled automatically if we use ```with```.
If we add ```try exception```, it would be good because we can handle exception errors when opening or writing.
```
def save_badge (path, badge):    
    try:    
        with open(path, "w") as f:    
            f.write(badge)    
    except IOError as e:    
        print("Failed to save badge:", str(e)) 
```

Reply:
Thanks, I applied your suggestion.